src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -43,7 +43,7 @@ public static ProcessInfo[] GetProcessInfos(string machineName)
         /// <summary>Gets an array of module infos for the specified process.</summary>
         /// <param name="processId">The ID of the process whose modules should be enumerated.</param>
         /// <returns>The array of modules.</returns>
-        internal static ModuleInfo[] GetModuleInfos(int processId)
+        internal static unsafe ModuleInfo[] GetModuleInfos(int processId)
```

Comment:
Rather than adding this to the signature here and below, can you instead just scope it to the statement that needs it?